docusaurus.config.js

```diff
@@ -148,6 +148,13 @@ const config = {
         additionalLanguages: ['csharp'],
       },
     }),
+    scripts: [
+      {
+        src:
+          '/js/loadposthog.js',
+        async: false,
```

Comment:
i'm assuming that blocking the rendering while this script loads is intentional ?

Reply:
Yup! This is what is recommended for proper logging of events.